c2corg_ui/templates/article/view.html

```diff
@@ -0,0 +1,143 @@
+<%!
+from c2corg_common.attributes import activities, image_categories, image_types
+from c2corg_ui.templates.utils import get_lang_lists
+from json import dumps
+%>
+
+<%inherit file="../base.html"/>
+<%namespace file="../helpers/common.html" import="show_title, show_fulldate"/>
+<%namespace file="helpers/detailed_article_attributes.html" import="get_article_general"/>
+<%namespace file="../helpers/view.html" import="get_comments, get_image_gallery, photoswipe_gallery,
+  get_document_description, get_document_locale_text, show_attr, show_missing_langs_links, show_other_langs_links,
+  show_archive_data, get_article_activities, show_areas, show_float_buttons, show_route_title,
+  show_associated_waypoints, show_associated_outings, show_associated_routes,
+  delete_association_confirmation_modal, associated_images_featurelist, show_badge"/>
+
+<%
+article_id = article['document_id']
+article['doctype'] = 'articles'
+other_langs, missing_langs = get_lang_lists(article, lang)
+%>
+
+<%block name="pagelang">lang="${lang}"</%block>
+<%block name="pagetitle"><title>${show_route_title(locale)}</title></%block>
+<%block name="metarobots">
+  % if version:
+    <meta name="robots" content="noindex,follow">
+  % else:
+    <meta name="robots" content="index,follow">
+  % endif
+</%block>
+
+<%block name="moduleConstantsValues">
+  module.value('documentData', {
+    "document_id": ${article.get('document_id')},
+    "lang": "${lang}",
+    "type": "c",
+    "topic_id": ${dumps(locale.get('topic_id'))}
+    % if not version:
+       , "associations": ${dumps(article.get('associations')) | n}
+    % endif
+  });
+</%block>
+
+<header class="view-details-title">
+  <h1 class="routes">
+##     <div class="icons-header routes">${get_article_activities(article, 'top')}</div>
+    <span class="title">${show_route_title(locale)}</span><br>
+    ${show_badge(article, 'article')}
+  </h1>
+</header>
+
+<section class="view-details-section images" app-view-details>
+
+  % if version:
+    ${show_archive_data('articles', article, locale, version)}
+  % endif
+
+  <div class="view-details-informations col-xs-12  informations">
+    <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#document-informations">
+      <span translate>Informations</span><span class="glyphicon glyphicon-menu-down"></span>
+    </h3>
+    <section id="document-informations" class="collapse in">
+      ${get_article_general(article)}
+      % if not version:
+##         ${get_article_participants(outing)}
+      % endif
+    </section>
+  </div>
+
+  <div class="description">
+    ${get_document_description(locale, 'article')}
+##     ${get_document_locale_text(locale, 'description')}
+  </div>
+
+  ## ASSOCIATIONS
```

Comment:
> We should rather use the associations rendering of waypoints or routes.

Mmmm perhaps it's already the case, since there's another association block a few lines below.

Anyway this "areas-associations"-like block is not relevant here. Please remove it.